src/sorcha/modules/PPConfigParser.py

```diff
@@ -639,6 +639,27 @@ def PPConfigFileParser(configfile, survey_name):
     else:
         config_dict["rng_seed"] = None
 
+    try:
+        config_dict["lightcurve"] = config.getboolean("EXPERT", "lightcurve", fallback=False)
+    except ValueError:
+        pplogger.error(
+            "ERROR: could not parse value for lightcurve as a boolean. Check formatting and try again."
+        )
+        sys.exit(
+            "ERROR: could not parse value for lightcurve as a boolean. Check formatting and try again."
+        )
+
+    config_dict["lc_model"] = config.get("EXPERT", "lc_model", fallback=None)
+    config_dict["lc_model"] = None if config_dict["lc_model"] == "None" else config_dict["lc_model"]
+
+    if config_dict["lightcurve"] and not config_dict["lc_model"]:
```

Comment:
Would it make sense to simplify this logic? i.e. `config_dict['lc_model'] == None` implies that `config_dict['lightcurve'] == False`, otherwise True?